copy/src/App.jsx

```diff
@@ -0,0 +1,68 @@
+import { useEffect, useState } from "react";
+import { Card } from "./component/Card/card.jsx";
+import { Header } from "./component/Header/header.jsx";
+import { Footer } from "./component/Footer/footer.jsx";
+import { SearchBar } from "./component/SearchBar/searchbar.jsx";
+import defaultImage from "./card-component-default.png";
+import "./style.css";
+
+const getFetch = async () => {
+  const response = await fetch(
+    "https://bootcamp-api.codeit.kr/api/sample/folder"
+  );
+  const body = await response.json();
+  return body;
+};
```

Comment:
```suggestion
const getFetch = async () => {
  const response = await fetch(
    "https://bootcamp-api.codeit.kr/api/sample/folder"
  );
  return response.json();
};
```
[중요도: 높음] 함수명이 명시적이지 않네요... 정확한 용도를 파악하기 어려운 이름은 좋지 않습니다.
[중요도: 높음] response.json()에는 await를 붙일 필요가 없습니다. json화 시키는 것은 비동기 작업이 아닙니다.
[중요도: 중간] 굳이 body라는 변수를 선언하고 return할 필요는 없어보이는군요,...!